core/src/main/java/edu/ucr/cs/riple/core/Config.java

```diff
@@ -559,10 +577,11 @@ public void write(Path path) {
         downstreamDependency.put("BUILD_COMMAND", downstreamBuildCommand);
       }
       json.put("DOWNSTREAM_DEPENDENCY_ANALYSIS", downstreamDependency);
-      try (FileWriter file = new FileWriter(path.toFile())) {
+      try (BufferedWriter file =
+          Files.newBufferedWriter(path.toFile().toPath(), Charset.defaultCharset())) {
         file.write(json.toJSONString());
       } catch (IOException e) {
-        e.printStackTrace();
+        System.err.println("Error happened in writing config json, " + e);
```

Comment:
Why coma instead of `:`?

Reply:
That was a mistake, thanks for noticing this.
e33fde946908b14e6037a5084657397176de1c36